packaging/hudi-flink-bundle/pom.xml

```diff
@@ -141,6 +141,13 @@
 
                   <include>org.apache.hbase:hbase-common</include>
                   <include>commons-codec:commons-codec</include>
+                  <include>org.apache.spark:spark-sql_${scala.binary.version}</include>
```

Comment:
Must we need to include spark dependencies into flink's bundle?

Reply:
If we have enable the hive sync for flink, we should include the spark dependencies. Because currently HiveSyncTool need the spark dependencies to generate the spark table properties.  
One thing I can improve here is making the scope of the spark dependencies to `${flink.bundle.hive.scope}`.

Reply:
@danny0405 Any thoughts that we can use to optimize? IMO, it seems to be not very graceful.

Reply:
well I think i can remove the spark dependencies by writing a util to convert the parquet schema to spark schema json without the spark lib.  Maybe it need some more work to do.